### PR TITLE
Lvpyt 58-61 Refactoring code and tests

### PR DESCRIPTION
**Pull request by vova-kul**
Tests with coverage 100%

### CODE REVIEW

**Comment by vova-kul** [comment]
retest this please

File: itaplay/itaplay/company/models.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.contrib.auth.models import User
```
Comment (lhalam) [inline itaplay/itaplay/company/models.py]: delete empty line

File: itaplay/itaplay/tests/company/test_models.py [hunk 2]
```diff
@@ -113,40 +113,40 @@ def test_set_company_create(self):
...
         company = Company.get_company(1)
-        self.assertEqual(company.company_name, "testcompany1")
-        self.assertEqual(company.company_mail, "test@test.test1")
-        self.assertEqual(company.company_phone, "+380951234567")
-        self.assertEqual(company.company_zipcode, "379007")
+        self.assertEqual(company.name, "testcompany1")
+        self.assertEqual(company.mail, "test@test.test1")
+        self.assertEqual(company.phone, "+380951234567")
+        self.assertEqual(company.zipcode, "379007")
         self.assertEqual(company.administrator, admin)
```
Comment (lhalam) [inline itaplay/itaplay/tests/company/test_models.py]: delete space

Comment (vova-kul) [reply itaplay/itaplay/tests/company/test_models.py]: Done

File: itaplay/itaplay/company/models.py (hunk 1 above)
Comment (vova-kul) [reply itaplay/itaplay/company/models.py]: Done

**Pull request merged**